LinaTheExplorer/LinaTheExplorer/2. Hi bye device/SelectedCars.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hi_bye_device
+{
+    class SelectedCars
+    {
+        public static List<Car> SelectCars(List<Car> listOfCars)
```

Comment:
good, I see you implemented car filtering here 👍 
Its better than it was before